status-server/src/public/ClientPage.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Example Side by Side client</title>
+  <link rel="stylesheet" href="./style/client.css">
+</head>
+<body>
+  <h1>Example status client</h1>
+
+  <section class="container" id="visualization">
```

Comment:
I'm not sure why we need this on the page? Shouldn't this just be a couple of iframes with the canvas, script, buttons etc in the iframed page?